### PR TITLE
{tools}[GCCcore/13.3.0] p7zip v17.05

### PR DESCRIPTION
**Pull request by maxim-masterov**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/p/p7zip/p7zip-17.05-GCCcore-13.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+%(installdir)s/libexec/7z $@' > %(installdir)s/bin/7z""",
+    "chmod +x %(installdir)s/bin/7z",  # set execution bits according to current umask
+]
+files_to_copy = [
+    (['bin/7za', 'bin/7zr', 'bin/7zCon.sfx'], 'bin'),  # stand-alone binaries
+    (['bin/7z', 'bin/7z.so', 'bin/Codecs'], 'libexec'),
+]  # 7z requires 7z.so plugin in same directory
+
+sanity_check_paths = {
+    'files': ['bin/7z', 'bin/7za', 'bin/7zCon.sfx', 'bin/7zr', 'libexec/7z', 'libexec/7z.so'],
```
[the remark this answers is not in the document]
Comment (maxim-masterov) [reply easybuild/easyconfigs/p/p7zip/p7zip-17.05-GCCcore-13.3.0.eb]: missed it, thanks!

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ jsc-zen3

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ generoso

**Review by SebastianAchilles — APPROVED** [review]
lgtm

**Comment by SebastianAchilles** [comment]
Going in, thanks @maxim-masterov!

**Pull request merged**